hcl/hcl.go

```diff
@@ -320,7 +341,9 @@ func ProductsMap(products []*Product) map[string]*Product {
 	return m
 }
 
-func mapRedacts(redactions []Redact) ([]*redact.Redact, error) {
+// MapRedacts maps HCL redactions to "real" `redact.Redact`s
+// 	This is a public function so it can be used to map redactions from the Agent and Products
```

Comment:
I don't know that the comment about the purpose of this being a public function adds much to understanding the code. Although we don't (yet) generate go docs for hcdiag, I also think this formatting might make its documentation look a little odd too. Just my $.02.

Reply:
That one was a note-to-self that got left; removed!